README.md

```diff
@@ -32,9 +32,13 @@ pipenv install
 
 ## Usage
 
-- Run `pipenv run python serialize_data.py` to generate JSON file named `endpoints_data.json`. Optionally, you can specify the path of the proposed endpoints docs  using `--file-path` option like:
+- Run `pipenv run python serialize_data.py` to generate JSON file named `endpoints_data.json`. Optionally, you can specify the path of the proposed endpoints docs  using `--file-path` or `-f` option like:
 
-    ```pipenv run python serialize_data.py --file-path /pat/to/endpoints_data.json```
+    ```pipenv run python serialize_data.py --file-path /path/to/proposed_endpoints.md```
+
+    You can also provide multiple files as
+
+    ```pipenv run python serialize_data.py -f /path/to/proposed_endpoints.md -f path/to/another_proposed_endpoints.md```
```

Comment:
The docs does not have any mention for running like this:

     ```pipenv run python serialize_data.py *.md ```

You've this format mentioned in the docstring though 🤔

Reply:
Updated in the latest commit.